### ISSUE TITLE
A11Y sort functionality

### ISSUE THREAD

**Issue by SarahMagdalena** [daisy/ace-gui#28]
It seems to me like the A11Y sort functionality in the metadata view doesn't work properly... 
I tried an ePub which has both dc metadata and accessibility metadata. The third column of the metadata table shows "Yes" for the accessibility metadata and is empty for the other ones. 
When pressing A11Y sort, I would expect the entries to be sorted according to this entry, but the order actually doesn't change.

**Comment by rdeltour** [comment]
I'll look into it, thanks for the report!
